### PR TITLE
create Games Controller and Model

### PR DESCRIPTION
**Pull request by saroar**
Hey i just generate Games controller and model hope it will review soon :+1:

### CODE REVIEW

**Comment by radiofreemattd** [comment]
There are a few spelling errors that you might want to check (see, e.g. line #3 of your controller where you spell `games` differently than in other places.

Also, do we need the JSON support at this point?

Also, and this is just a personal issue, I would delete everything that we don't absolutely need, which is really just controller file itself, and perhaps view placeholders for each of the actions.

Just personal opinions... :)

**Comment by saroar** [comment]
thanks let me solve it :)

**Comment by radiofreemattd** [comment]
You probably want to fix the spelling of "games" in `games_controller.rb`. Others might not care, but I would rip out any files that were generated by the generator but that we are not specifically using. So, for instance, the css and coffeescript files in `assets`. We haven't decided on a testing strategy or factories/fixtures, so I would pull all that out. Literally, I think all that you need is the controller file and the view files.

Again, totally personal preference and more steeped in OCD than probably an actual technical need to keep the codebase slim.

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class GamesController < ApplicationController
+  def index
+    @gamnes = Games.all
```
Comment (jonmast) [inline app/controllers/games_controller.rb]: This looks like a typo.

**Comment by jonmast** [comment]
I would vote for removing the fixtures and using FactoryGirl as well.

**Comment by radiofreemattd** [comment]
At this point, there have been enough changes that I think the best thing to do is to close this request, ask you to pull the latest code from master, and re-submit the pull request. Apologies in advance if that is a hassle, but there are things here that don't make sense (e.g. the pull request shows the removal of JSON as a deletion, when it is not in master). I think all we need is the controller file, the model, the changes to the schema and the routing file and the view files...anything to do with front-end assets, helpers, or testing should be deleted IMO.

Pull the latest code so that you have the latest and greatest and then re-submit, yeah? Again, sorry, but I think that will be cleaner and easier for all to see. Fair?

**Comment by saroar** [comment]
Dont need sorry it ok and this is part of our job :) specially my job call CRUD lol :)

**Pull request closed without merging**